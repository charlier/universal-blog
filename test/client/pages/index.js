import { h } from 'preact';
import { shallow } from 'preact-render-spy';

import Index from '../../../src/client/pages';

describe('Index Page', () => {
  it('Renders the Index Page', () => {
    const wrapper = shallow(<Index />);
    expect(wrapper).toMatchSnapshot();
  });
});
