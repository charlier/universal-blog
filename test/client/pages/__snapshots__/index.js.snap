// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page Renders the Index Page 1`] = `
preact-render-spy (1 nodes)
-------
<UnnamedComponent0>
  <p>Main page</p>
</UnnamedComponent0>

`;
